documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,150 @@
+
+If WCA Regulations are not correctly observed during a competition, results may be modified at the discretion of the WCA Delegate, WCA Regulations Committee, WCA Results Team, WCA Disciplinary Committee, or the WCA Board.
+
+### Announcement Criteria
+The WCA Delegate must submit the following information when requesting approval of a competition:
+
+1. Competition name, nickname, and ID
+   1. The name must not be too general. It should be related in some way to the competition itself, such as:
+      1. The location of the competition
+      2. The competition venue
+      3. The season or date of the competition
+      4. The events of the competition
+      5. The convention the competition is a part of
+      6. A local cultural reference
+      7. A sponsor of the competition
+   2. Capitalization of the competition name must be appropriate. The first letter of each word in the competition’s name should be capitalized and subsequent letters within words should be lowercase, except articles and prepositions, which should be always lowercase. Acronyms should be capitalized.
+   3. The name must end with the year of the last day of the competition.
+   4. The competition name must be in Roman-readable letters and may only contain characters from A-Z, a-z, 0-9, or similar characters. Only '.:-& are permitted as special characters.
+   5. The competition name must be 50 characters or less, including whitespaces.
+   6. If the competition is named after anyone or anything, they must be persons or things lacking any controversial meaning worldwide.
+   7. Names should contain the location of the competition.
+   8. Competitions may not be named after a living cuber, competition organizer, or any competition staff.
+   9. Names must not contain possibly offensive parts or memes.
+   10. The name cannot contain website domains or anything susceptible to be interpreted as a link, hashtag or in general an electronic identifier.
+   11. The name cannot contain any trademarks unless the organization team is formally authorized, in writing, to use such names.
+   12. Names should not contain any unnecessary parts in the name, like "competition".
+   13. The competition nickname and ID must resemble the competition name.
+       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
+    14. Abbreviations and acronyms should be avoided unless the competition name is too long (more than 32 characters).
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+   3. Competitions will be accepted if there are no other competitions within 100 km driving distance and less than 19 days away.
```

Comment:
I'm not sure this is the best wording for this. Just because the proximity policy is adhered to doesn't mean that the competition will necessarily be accepted.

Reply:
I agree it reads as though that is the deciding requirement.